Makefile

```diff
@@ -17,5 +17,9 @@ README.html: README.md docs/metadata.yml
 	docs/metadata.yml README.md -o $@ \
 	--standalone --toc --toc-depth 2
 
+Documentation.txt: README.md docs/metadata.yml roles/*
```

Comment:
Also, depending on `roles/*` is probably not what you want (you want to depend on individual *files* inside the roles directory; see the `DEFAULTS` variable at the top of the `Makefile`).